packages/dev-plugin/src/utils.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+export function ensureTempDir(filePath: string): void {
```

Comment:
this function is named `ensureTempDir` but I don't see any temporary directory.
Additionally, it's strange to not handle the catch. Doing so, you are not ensuring things, you are just ignoring errors